starters/next-wordpress-starter/pages/pages/[...uri].jsx

```diff
@@ -35,11 +36,21 @@ export default function PageTemplate({ menuItems, page }) {
 	);
 }
 
-export async function getServerSideProps({ params: { uri }, res }) {
+export async function getServerSideProps({
+	params: { uri },
+	res,
+	preview,
+	previewData,
+}) {
 	const menuItems = await getFooterMenu();
-	const page = await getPageByUri(uri);
+	const credentials = getAuthCredentials();
```

Comment:
As mentioned in the other comment, can we move this call into the `getXPreview` function? Or is there a good reason to do it this way?